src/bootstrap/lib.rs

```diff
@@ -931,6 +939,24 @@ impl Build {
         self.package_vers(&self.release_num("rustfmt"))
     }
 
+    fn llvm_tools_vers(&self) -> String {
```

Comment:
any suggestion to get the LLVM version from source code rather than from a binary?

Reply:
LLVM version is set when [running `cmake` on it's source code](https://github.com/llvm-mirror/llvm/blob/a71e59ebf7ff06a6c39d28d6fea4f9340c499d0f/CMakeLists.txt#L26).
After `cmake` was run the version is coded into [`llvm/Config/llvm-config.h`](https://github.com/llvm-mirror/llvm/blob/4833be099d271d1102c6d7c1fcd63d870a92a85f/include/llvm/Config/llvm-config.h.cmake#L75).